sickbeard/notifiers/kodi.py

```diff
@@ -602,7 +602,4 @@ def update_library(self, showName=None):
 
             # needed for the 'update kodi' submenu command
             # as it only cares of the final result vs the individual ones
-            if result == 0:
-                return True
-            else:
-                return False
+            return bool(result)
```

Comment:
This is a bug.
Before it was only returning True if result is zero. Now it's the other way around.